src/esw/brushed_motors/ROSHandler.cpp

```diff
@@ -0,0 +1,59 @@
+#include "ROSHandler.h"
+
+// REQUIRES: rosNode is a pointer to the created node.
+// MODIFIES: n, subscribersByName, and publishersByName.
+// EFFECTS: Initializes all subscribers and publishers.
+void ROSHandler::init(ros::NodeHandle* rosNode) {
+
+    n = rosNode;
+
+    std::vector<subscriberData> openLoopSubscribers = {
```

Comment:
I wasn't sure whether to hard code the subscribers  and channels in the .h file or to do this but I think the current way looks cleaner tbh. I also could've put the scope of openLoopSubscribers inside the class and back into the .h file but didn't want to keep memory there for entire lifetime of program (but it's not the end of the world so if anybody has objections, i can put it inside the class instead)